ipapython/certmonger.py

```diff
@@ -307,17 +307,30 @@ def request_and_wait_for_cert(
     The method also waits for the certificate to be available.
     """
     reqId = request_cert(nssdb, nickname, subject, principal,
-                         passwd_fname, dns, ca, profile,
-                         pre_command, post_command)
-    state = wait_for_request(reqId, timeout=60)
+                         passwd_fname, dns, ca, profile)
+    state = wait_for_request(reqId, timeout=180)
     ca_error = get_request_value(reqId, 'ca-error')
     if state != 'MONITORING' or ca_error:
         raise RuntimeError("Certificate issuance failed")
+
+    # Add presave and postsave commands
+    # This is not done earlier to avoid running the cmds
+    # during the initial request
```

Comment:
Instead of this, I would rather not restart the services in `renew_ra_cert`, `restart_httpd` and `restart_dirsrv` if they were not previously running.